digitizer/submission.py

```diff
@@ -95,15 +95,17 @@ def get_zip_file(self):
                              compression=zipfile.ZIP_DEFLATED) as zhandle:
             for i, isotherm in enumerate(self):
                 directory = isotherm.json['DOI'].replace('/', '')
-                filename = '{d}/{d}.Isotherm{i}.json'.format(d=directory,
-                                                             i=i + 1)
-                zhandle.writestr(filename, isotherm.json_str)
 
                 if isotherm.figure_image:
                     filename = '{d}/{d}.Isotherm{i}_{f}'.format(
                         d=directory, i=i + 1, f=isotherm.figure_image.filename)
+                    isotherm.json['associated_content'] = [filename]
```

Comment:
So just to be sure, you think this is better?

If you prefer to set it this way, then perhaps remove the place where you set it to a different filename, otherwise we may get confused lateron

Reply:
Ah hm, I now saw your other comment... is this really making things easier?
I would tend to say it is easier to stick to the same filename in the json, whether it is downloaded as a zip or not.

Reply:
A little debate on this is worthwhile, as I'm trying to think of how users will use the panel. I wouldn't put this in the "making things easier" category.

Here's my reasoning, based on some test usage on my end: (I'm currently using the panel to create JSON isotherms for a paper that is not published yet.)

1. Starting with a clean form, fill in all the entries in the form
2. Attach a graphic file to it
3. Click "Plot Data"

At this point in the process, a user could take two different actions: either "Download data.json" or "Add to submission." 

* In the "Download data.json" route, the present script preserves the filename from the attached graphic. (Even though the attached graphic is not downloaded with data.json [though I guess we could add that feature].) I can anticipate users creating JSON files, downloading them one by one (for a variety of reasons), and assembling the package of JSON files and associated graphics manually. Changing the filename reference in the JSON data would confuse this usage.

* In the "Add to submission" route, the filename is changed and the reference in the JSON payload is updated accordingly when the ZIP file is produced. I'll readily admit that this would be the more elegant, "better" usage of the panel. Since the files are organized together, it makes a lot of sense to rename files and update internal references automatically.

An alternative is to not rename the graphic files, but I like that feature as it organizes the files for the user (which I'm sure _most_ people including me should do).

Reply:
Ok. Let's follow your suggestion for the moment and see how people use the tool.
We can still decide to amend this later